### DISCUSSION TITLE
Failing to connect

### DISCUSSION THREAD

**Discussion by a4ginatl** [codefori/vscode-ibmi#2176] [IBM i] [answered]
I have codefori installed on my system and it was working. I have not used this system for months and now need to use it again. When I try to log on, it fails with "check your credentials (All configured authentication methods failed.

I do not remember how I configured the connection. Am I using a password or a private key?

I changed the password and tried that, but it failed.

I don't know if I used a private key.

Anyway, how can I reset it and get it working again?

Thank you,
Darryl Freinkel.

**Comment by chrjorgensen** [comment, marked as answer]
@a4ginatl You can remove all stored credentials (password + path of private key) on the connection by right-click and choose `Login Settings` and click the button `Remove auth methods` at the bottom of the panel:

![billede](https://github.com/user-attachments/assets/7ad716fc-bf8a-4158-a5ec-ccb4c5308931)

Another troubleshooting step is to use another SSH client, e.g. [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/), to try the login with manually entered credentials...

**Comment by a4ginatl** [comment]
Thank you. I connected.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562
